runtime/src/main/java/org/corfudb/recovery/FastObjectLoader.java

```diff
@@ -232,20 +254,53 @@ public void updateStreamTails(long address, ILogData logData) {
      * @param entry entry to potentially apply
      * @return if we need to apply the entry.
      */
-    private boolean shouldEntryBeApplied(UUID streamId, SMREntry entry) {
+    private boolean shouldEntryBeApplied(UUID streamId, SMREntry entry, boolean isCheckpointEntry) {
+        // If white list mode, ignore everything that is not in the list or checkpoints
```

Comment:
The comment is incorrect?

Reply:
is poorly written.  I'll rephrase it. What I wanted to emphasize was that in this list there are both the streams passed by the client and their derived checkpoint streams.